packages/beacon-node/src/chain/stateCache/datastore/file.ts

```diff
@@ -13,9 +13,10 @@ const CHECKPOINT_FILE_NAME_LENGTH = 82;
 export class FileCPStateDatastore implements CPStateDatastore {
   private readonly folderPath: string;
 
-  constructor(parentDir = ".") {
-    // by default use the beacon folder `/beacon/checkpoint_states`
-    this.folderPath = path.join(parentDir, CHECKPOINT_STATES_FOLDER);
```

Comment:
oh yes, this is definitely wrong, can you re-open this branch against unstable please

Reply:
https://github.com/ChainSafe/lodestar/pull/7551